pyuvdata/tests/test_uvbeam.py

```diff
@@ -1224,10 +1146,20 @@ def test_healpix():
 
     # repeat for efield beam
     efield_beam = UVBeam()
-    efield_beam = fill_dummy_beam(efield_beam, 'efield', 'healpix')
+    efield_beam.read_cst_beam(cst_files[0], beam_type='efield', frequency=150e6,
+                              telescope_name='TEST', feed_name='bob',
+                              feed_version='0.1', feed_pol=['x'],
+                              model_name='E-field pattern - Rigging height 4.9m',
+                              model_version='1.0')
+    # generate more frequencies for testing by copying and adding
+    new_beam = copy.deepcopy(efield_beam)
+    new_beam.freq_array = efield_beam.freq_array + efield_beam.Nfreqs * 1e6
+    efield_beam += new_beam
+
+    efield_beam.az_za_to_healpix()
     old_history = efield_beam.history
 
-    freqs_to_keep = efield_beam.freq_array[0, np.arange(31, 56)]
+    freqs_to_keep = efield_beam.freq_array[0]
```

Comment:
Why is this no longer downselecting?

Reply:
It is downselecting to only the first frequency. This is a consequence of getting the beams from files rather than generating them with random numbers -- now there are only 2 frequencies so I'm downselecting to one of them.